lib/cosmos/dart/lib/dart_constants.rb

```diff
@@ -0,0 +1,15 @@
+# encoding: ascii-8bit
+
+# Copyright 2018 Ball Aerospace & Technologies Corp.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+
+# Implement constants common to DART (Data Archival Retrieval and Trending).
+module DartConstants
+  # @return [Integer] maximum number of results returned from a decommutation query
+  MAX_DECOM_RESULTS = 10000
```

Comment:
This seemed to be sprinkled everywhere so I made it a constant. I initially put it in Common but that didn't work due to it bringing in the Rails models.